safe/reportv4/extractors/minimum_needs.py

```diff
@@ -59,12 +60,18 @@ def minimum_needs_extractor(impact_report, component_metadata):
             if need_parameter.unit.abbreviation:
                 header = '%s [%s]' % (
                     header, need_parameter.unit.abbreviation)
+            # We don't have any layer to store minimum needs for now.
+            # So, for now just calculate it manually
             need_index = analysis_layer.fieldNameIndex(
-                need_parameter.name)
+                total_affected_field['field_name'])
```

Comment:
Need to consult with @Gustry and @timlinux .
Do we store minimum needs calculations on a layer (for example in analysis layer).
Or, we just calculate minimum needs in the reporting (possible by total affected * needs_parameter.value).

For myself, I prefer storing the calculations on a layer, so reporting only handled presentation logic.